### PR TITLE
Login should be required for any page of the app

### LINKED ISSUE #186: Login should be required for any page of the app
User is able to access to pages of the system without login as an valid user first:
http://127.0.0.1:5000/contact/dashboard<br>
http://127.0.0.1:5000/page/dashboard<br>
http://127.0.0.1:5000/theme/

### PR DESCRIPTION
**Pull request by memoglr**
Fixes #186

### CODE REVIEW

File: shopyo/modules/contact/view.py [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 
 @contact_blueprint.route("/")
+@login_required
 def index():
     context = base_context()
     form = ContactForm()
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/contact/view.py]: Login required not required here as public should be able to contact

File: shopyo/modules/page/view.py [hunk 2]
```diff
@@ -38,6 +39,7 @@
 
 
 @module_blueprint.route("/")
+@login_required
 def index():
     context = base_context()
     pages = Page.query.all()
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/page/view.py]: Similarly here the public should be able to see all pages available to read

File: shopyo/modules/page/view.py [hunk 3]
```diff
@@ -47,6 +49,7 @@ def index():
 
 
 @module_blueprint.route("/<page_id>/<slug>")
+@login_required
```
Comment (Abdur-rahmaanJ) [inline shopyo/modules/page/view.py]: Here also the public should be avle to see individual page. Else PR ok

**Comment by memoglr** [comment]
Changes made.

**Comment by Abdur-rahmaanJ** [comment]
Great!

**Pull request merged**